spring-cloud-kubernetes-fabric8-config/src/main/java/org/springframework/cloud/kubernetes/fabric8/config/reload/ConfigReloadDefaultAutoConfiguration.java

```diff
@@ -1,182 +0,0 @@
-/*
- * Copyright 2013-2020 the original author or authors.
```

Comment:
I deleted this one. It was not used in `spring.factories`, so I assume it was some kind of an artifact of previous changes. We have a major release, so imho: this is safe.